pipelines/services/types.go

```diff
@@ -0,0 +1,89 @@
+package services
+
+import "time"
+
+type PipelineRunStatusResponse struct {
+	TotalCount int         `json:"totalCount,omitempty"`
+	Pipelines  []Pipelines `json:"pipelines,omitempty"`
+}
+
+type StaticPropertyBag struct {
+	TriggeredByUserName    string `json:"triggeredByUserName,omitempty"`
+	SignedPipelinesEnabled bool   `json:"signedPipelinesEnabled,omitempty"`
```

Comment:
The combination of booleans with `omitempty` is troublesome - default true values may not return from the server, but the value may be false.
```suggestion
	SignedPipelinesEnabled *bool   `json:"signedPipelinesEnabled,omitempty"`
```